largescale/config/million-collection-testing

```diff
@@ -8,4 +8,4 @@ collections=1000000
 collection_rampsize=4000
 ramp_interval=300
 working_set_docs=40000000
-fail_at_ms=50
+fail_at_ms=5
```

Comment:
Is the intent to leave this value lower, or should it be set back to the original now that you have validated failure modes? Similarly for `fail_at_throughput_factor`

Reply:
The tuning for `fail_at_ms` and `fail_at_throughput_factor` is to achieve testing efficiency by making test fail faster, and fail with desired reason - e.g. if leaving `fail_at_ms` as default value `50`, test will fail by average throughput. 

And since you are ok with the other changes, I will go ahead changing the `fail_at_ms` and `fail_at_throughput_factor` values back to default.